GOVERNANCE.md

```diff
@@ -0,0 +1,75 @@
+These changes will be integrated into the main branch of the code upon approval. Consistent 
+contributions also provides the opportunity for Contributors to be eligible to join as a *Maintainer*, the final approval of which will require a vote.
+
+***Maintainer***: These are the core team who have "merge" permissions on the repo and are working to keep the code sustainable long term.  
+
+To maintain their active status as a User, Contributor, or Maintainer, all members will agree to adhere to a Code of Conduct.
+
+## Memorandum of Understanding
+
+In addition to our Community Code of Conduct, all code Maintainers will sign a memorandum of understanding (MoU) 
+that will describe general expectations and responsibilities. The MoU that will serve as part of the `Phoebus` Community requires the following:
+
+- Maintainers will uphold and exemplify the _Development Model_ outlined as part of the `phoebus` Community described [here](CONTRIBUTING.md).
+- Maintainers will make a good-faith effort to attend team meetings. In an effort to accommodate the dynamic schedules and availability of many of the core team of *Maintainers*, coordination efforts will be made to ensure that *at least* 1 *Maintainer* is present for all meetings. 
+- Maintainers will make a good-faith effort to attend the annual workshop.
+- Maintainers will make a good-faith effort to monitor the fraction of the meeting time that they speak. The meeting time is limited, and several people may have something interesting to say, report on their updates, or contribute constructive arguments. One should be self-aware and notice if one is dominating conversations, and attempt to make a conscious effort to include other voices and perspectives in the discussion.
+- Maintainers consider it their responsibility to voice their questions and/or concerns as early as possible.  This will ensure that the organization addresses concerns rapidly and effectively.
+- Maintainers consider it their responsibility to listen and understand questions or concerns raised by other partners or through concerns communicated through email to individual *Maintainers*.
+
+
+### Approval by Vote:  
+Many of the decisions in the organization will be enacted after an approval by majority vote of the Maintainers. 
+The process of listening and adjusting strategies to meet everyone's needs is vital for power-sharing organizations.  
+Importantly, spending a little time up front to listen and resolve concerns reduces the pain and time spent later in intense conflict resolution.  
+
+### Communication and Connection: 
+To facilitate open and effective communication, the `Phoebus` will offer multiple levels of communication.  
+Developers will communicate via email, GitHub, and a `Phoebus` Slack channel. 
+
+To respond to user inquiries, the Phoebus Project will host a mailing list via Google or similar.  
```

Comment:
Who's going to start the google mailing list? :)

Reply:
I put it on my todo list. I want a service that isn't google, but probably it will be google..